README.md

```diff
@@ -266,69 +266,116 @@ namespace: reloader
 
 Alternatively if you have configured helm on your cluster, you can add Reloader to helm from our public chart repository and deploy it via helm using below-mentioned commands. Follow [this](docs/Helm2-to-Helm3.md) guide, in case you have trouble migrating Reloader from Helm2 to Helm3.
 
-```bash
+#### Installation
+
+```yaml
```

Comment:
keep this as bash, it is not a yaml file but commands you execute in a shell

Reply:
Thanks, I have updated my PR.